pycontrails/core/models.py

```diff
@@ -553,11 +570,21 @@ def set_source_met(
             # take the first var name output from ensure_vars
             met_key = self.met.ensure_vars(var)[0]
 
+            q_method = self.params["interpolation_q_method"]
+
             # interpolate GeoVectorDataset
             if isinstance(self.source, GeoVectorDataset):
-                interpolate_met(self.met, self.source, met_key, **self.interp_kwargs)
+                interpolate_met(
+                    self.met, self.source, met_key, q_method=q_method, **self.interp_kwargs
```

Comment:
You could remove `q_method` from this interface if it were in `interp_kwargs`

Reply:
fe10a1df11aed5cedc1afe767701ef1a869b7059